imports/plugins/core/payments/server/no-meteor/queries/availablePaymentMethods.js

```diff
@@ -0,0 +1,18 @@
+import { paymentMethods } from "/imports/plugins/core/core/server/no-meteor/pluginRegistration";
+
+/**
+ * @name availablePaymentMethods
+ * @method
+ * @memberof Payments/NoMeteorQueries
+ * @summary TODO: query the Groups collection and return a MongoDB cursor
+ * @param {Object} context - an object containing the per-request state
+ * @param {String} shopId - shop ID for which to get payment methods
+ * @return {Object[]} TODO: Groups collection cursor
+ */
+export default async function (context, shopId) {
+  const shop = await context.queries.shopById(context, shopId);
+  const paymentMethodNames = Object.keys(paymentMethods);
+
+  return shop.availablePaymentMethods
```

Comment:
Change to `return (shop.availablePaymentMethods || [])`. Otherwise I get errors running the query until we make the migration to set this array.